### PR TITLE
Update event view page

### LINKED ISSUE #55: Update AppExternalToggle and Calendar layout on Event View Page
# Task 🔨

> What do I have to do? Be as specific as possible. Refer to the [Launch Pad docs](https://docs.ubclaunchpad.com/handbook/project-management/sprints#managing-tasks) for more information on tickets.

AppExternalToggle needs to call a function to switch calendars when clicked. AppExternalToggle needs its CSS rewritten to adhere to designs. It needs to be made as reusable as possible so other components in the app can use it. The calendar needs to have CSS updated. The 'save response' and 'copy event link' buttons should not change to vertical orientation when window is shrunk.

# Related MVP Tasks ⚙️ 

> Which MVP tasks does this accomplish? You can view all the tasks on the [notion page](https://www.notion.so/cc1e5567317a4591a016c73e405c3ae0?v=c5603d174e77451b9b027f9818e9c513).

[Create event](https://www.notion.so/Team-5-Priorities-and-Looking-Forwards-af1723b780354629a9a7d43d61896d46#68a3e4b345f24b0087820dd26dfed4ed)

# Related Wireframes ✏️ 

> Is any design work useful for this task? If so, attach a screenshot here.

_Figma: https://www.figma.com/file/P8cQy1qJhyDX4oTm1DxMtz/When3Meet?node-id=0%3A1_

# Related Issues ⚠️ 

> Any dependent/related issues would go here. Type in the issue like so: `#<number>`

### PR DESCRIPTION
**Pull request by baojill**
## Overview ⚙️

> Updated events view page for CSS that adheres to designs and better responsiveness

## Change Summary

* AppToggleExternalText will now call an update function when clicked so that Events.vue can switch between user calendar and group calendar (still need to write the function)
* AppToggleExternalText now adheres to designs
* 'Save Response' and 'Copy Event Link' buttons no longer stack on top of each other when window is shrunk
* Widened Calendar margins for better layout

## Screenshots

Screen Shot 2020-11-20 at 4 51 08 PM

## Related Issues ⚠️

> Put any issue that this PR addresses here. Make sure you use a [closing keyword](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) so GitHub knows to link the issue(s).

Closes #55

### CODE REVIEW

File: frontend/src/components/AppToggleExternalText.vue [hunk 1]
```diff
@@ -1,179 +1,123 @@
...
-  position: absolute;
-  display: block;
-  height: 18px;
-  width: 18px;
-  border-radius: 50%;
-  left: 0;
-  top: -3px;
-  transform: translateX(0);
-  transition: all 0.25s cubic-bezier(0.5, -0.6, 0.5, 1.6);
+.leftWhite {
```
Comment (tnwin) [inline frontend/src/components/AppToggleExternalText.vue]: rename to `.leftOff` or `.leftDisabled`

File: frontend/src/components/AppToggleExternalText.vue [hunk 2]
```diff
@@ -1,179 +1,123 @@
...
+  flex-shrink: 0;
+  margin-left: 3rem;
+  color: rgb(48, 48, 48);
 }
-
-/* Toggle Left */
-.toggle__button .toggle__switch::after {
-  background: #6791f0;
-  box-shadow: 0 0 1px #666;
+.rightWhite {
```
Comment (tnwin) [inline frontend/src/components/AppToggleExternalText.vue]: rename to `.rightOff` or `.rightDisabled`

**Review by tnwin — CHANGES_REQUESTED** [review]
* placed in separate component `AppToggleInternalText.vue`
* rename `White` to `Off` or `Disabled`

File: frontend/src/components/AppToggleExternalText.vue (hunk 2 above)
Comment (jillxi) [reply frontend/src/components/AppToggleExternalText.vue]: done!

**Review by tnwin — APPROVED** [review]

**Review by michelleykim — APPROVED** [review]

**Pull request merged**